packages/nimble-components/src/anchor-tabs/index.ts

```diff
@@ -0,0 +1,286 @@
+                event.preventDefault();
+                anchor = this.getTabAnchor(event.target as AnchorTab);
+                anchor.focus();
+                anchor.dispatchEvent(
+                    new KeyboardEvent('keydown', {
+                        key: event.key,
+                        bubbles: false
+                    })
+                );
+                break;
+            default:
+            // do nothing
+        }
+    };
+
+    private focusFirstOrLast(focusLast: boolean): void {
+        const focusableTabs = this.tabs.filter(t => !this.isDisabledElement(t));
+        const focusableIndex = focusLast ? focusableTabs.length - 1 : 0;
+        const index = this.tabs.indexOf(focusableTabs[focusableIndex]!);
+        if (index > -1) {
+            this.focusTabByIndex(this.tabs, index);
+        }
+    }
+
+    private readonly adjustForward = (): void => {
+        const group: HTMLElement[] = this.tabs;
+        let index = 0;
+        const focusedTab = group.find(x => x === document.activeElement);
+
+        index = focusedTab ? group.indexOf(focusedTab) + 1 : 1;
+        if (index === group.length) {
+            index = 0;
+        }
+
+        while (index < group.length && group.length > 1) {
+            if (this.isFocusableElement(group[index]!)) {
+                this.focusTabByIndex(group, index);
+                break;
+            } else if (focusedTab && index === group.indexOf(focusedTab)) {
+                break;
+            } else if (index + 1 >= group.length) {
+                index = 0;
+            } else {
+                index += 1;
+            }
+        }
+    };
+
+    private readonly adjustBackward = (): void => {
+        const group: HTMLElement[] = this.tabs;
+        let index = 0;
+        const focusedTab = group.find(x => x === document.activeElement);
+
+        index = focusedTab ? group.indexOf(focusedTab) - 1 : 0;
+        index = index < 0 ? group.length - 1 : index;
+
+        while (index >= 0 && group.length > 1) {
+            if (this.isFocusableElement(group[index]!)) {
+                this.focusTabByIndex(group, index);
+                break;
+            } else if (index - 1 < 0) {
+                index = group.length - 1;
+            } else {
+                index -= 1;
+            }
+        }
+    };
+
+    private readonly focusTabByIndex = (
+        group: HTMLElement[],
+        index: number
+    ): void => {
+        const focusedTab: HTMLElement = group[index]!;
+        focusedTab.focus();
+
+        this.tabs.forEach((tab: HTMLElement) => {
+            tab.setAttribute('tabindex', tab === focusedTab ? '0' : '-1');
+        });
+    };
+
+    private navigateToTab(index: number): void {
+        const tab = this.tabs[index] as AnchorTab;
+        this.getTabAnchor(tab).click();
+    }
+
+    private getTabAnchor(tab: AnchorTab): HTMLAnchorElement {
+        // eslint-disable-next-line @typescript-eslint/non-nullable-type-assertion-style
```

Comment:
Can we fix this lint error instead of suppressing it? There's good guidance about the rule [in its docs](https://typescript-eslint.io/rules/non-nullable-type-assertion-style/). Summary: replace ` as HTMLAnchorElement` with `!`.

Reply:
Done